tyk-single-dc/README.md

```diff
@@ -0,0 +1,208 @@
+## Tyk Tyk single dc
+
+`tyk-single-dc` provides the default deployment of a Tyk Single Data Center. It will deploy the data center components with the settings provided in the values.yaml file.
+
+It includes:
+- the Tyk Gateway, an open source Enterprise API Gateway (supporting REST, GraphQL, TCP and gRPC protocols)
+- Tyk Pump, an analytics purger that moves the data generated by your Tyk nodes to any back-end. Furthermore, it has all the required modifications to easily connect to Tyk Cloud or Multi Data Center (MDCB) control plane.
+- Tyk Dashboard, a license based component that allows you to interact with Tyk in a browser
+
+## Introduction
+
+By default, this chart installs following components as subcharts on a [Kubernetes](https://kubernetes.io/) cluster using the [Helm](https://helm.sh/) package manager.
+| Component | Enabled by Default | Flag |
+| --------- | ------------------ | ---- |
+|Tyk Gateway |true  | n/a                    |
+|Tyk Pump    |true | global.components.pump |
+|Tyk Dashboard| true| global.components.dashboard | 
+
+To enable or disable each component, change the corresponding enabled flag.
+
+Also, you can set the version of each component through `image.tag`. You could find the list of version tags available from [Docker hub](https://hub.docker.com/u/tykio).
+
+## Prerequisites
+
+* Kuberentes 1.19+
+* Helm 3+
+* Redis should already be installed or accessible by the gateway. For Redis installation instruction, please see https://tyk.io/docs/tyk-oss/ce-helm-chart/#recommended-via-bitnami-chart.
+
+## Installing the Chart
+
+To install the chart from Git repository in namespace `tyk` with the release name `tyk-single-dc`:
+
+    helm repo add tyk-helm https://helm.tyk.io/public/helm/charts/
+    helm repo update
+    helm show values tyk-helm/tyk-single-dc > values-single-dc.yaml --devel
```

Comment:
Please either use --devel for all commands or remove it..

Reply:
done